xml/Microsoft.VisualBasic.Compatibility.VB6/Support.xml

```diff
@@ -631,7 +631,7 @@
       </Parameters>
       <Docs>
         <param name="CurrentFont">A <see cref="T:System.Drawing.Font" />.</param>
-        <param name="Size">A <see langword="Single" /> that represents the Visual Basic 6.0 <see langword="FontSize" /> property.</param>
+        <param name="Size">A <see langword="T:System.Single" /> that represents the Visual Basic 6.0 <see langword="FontSize" /> property.</param>
```

Comment:
For types, `cref` should be used, not `langword`. The latter is reserved for keywords like `true`, `false` or `null`:
```suggestion
        <param name="Size">A <see cref="T:System.Single" /> that represents the Visual Basic 6.0 <see langword="FontSize" /> property.</param>
```